packages/appium/lib/bidi.ts

```diff
@@ -95,14 +100,25 @@ export function onBidiConnection(this: AppiumDriver, ws: WebSocket, req: Incomin
   }
 }
 
+function wrapCommandWithPlugins(driver: ExtensionCore, plugins: ExtensionCore[], method: string, params: StringRecord) {
```

Comment:
can we also have the return type declared?

Reply:
TS is smart enough to figure out the return type implicitly, is there a benefit in adding it explicitly?

Reply:
yes, it helps the linter to assert that the variable returned by function has the expected type